Jenkinsfile

```diff
@@ -11,9 +11,7 @@ buildMvn {
 
   doDocker = {
     buildJavaDocker {
-      publishMaster = true
-      healthChk = true
-      healthChkCmd = 'wget --no-verbose --tries=1 --spider http://localhost:8081/admin/health || exit 1'
+      publishMaster = 'yes'
```

Comment:
Is this gathered from some another spring repo, can you send link with example please

Reply:
This is typical settings for our modules
https://github.com/folio-org/mod-orders/blob/master/Jenkinsfile
https://github.com/folio-org/mod-data-export-spring/blob/master/Jenkinsfile